h5c/vic/src/vic-webapp/e2e/app.po.ts

```diff
@@ -22,9 +22,12 @@ export class VicWebappPage {
   private buttonComputeResource = 'button.cc-resource';
   private datacenterTreenodeCaret = 'button.clr-treenode-caret';
   private firstDcTreenodeCaretXpath = `(//button[contains(@class,'clr-treenode-caret')])[1]/clr-icon[contains(@dir, 'right')]`;
+  private basicAdvanceButton = 'a.btn.btn-link.pl-0';
```

Comment:
We need to make sure naming is consistent. Other places we have 'button..' while here we have 'basicAdvanceButton'...let's go with the latter format and have this 'buttonBasicAdvance'